src/server/middleware/render-index.js

```diff
@@ -91,6 +91,13 @@ export default function renderIndex(html, css, assetMap, store) {
       window.EXPERIMENTAL_TAGS=${process.env.EXPERIMENTAL_TAGS || false}
       window.EXPERIMENTAL_TEXTERUI="${process.env.EXPERIMENTAL_TEXTERUI || ""}"
       window.TWILIO_MULTI_ORG=${process.env.TWILIO_MULTI_ORG || false}
+      window.EXPERIMENTAL_PHONE_INVENTORY=${getConfig(
```

Comment:
one downside of using global window variables for these things is that it can't be per-org, then, because this globally loaded -- it might be better to have an organization graphql property that can return `getConfig('EXPERIMENTAL_PHONE_INVENTORY', organization)` and then condition on that (after adding to a (apparently not yet present) organization query in the components/AdminDashboard.jsx component